modules/sara_r4/cellular_r4_wrapper.c

```diff
@@ -132,8 +133,17 @@ CellularError_t Cellular_SocketSetSockOpt( CellularHandle_t cellularHandle,
                                            const uint8_t * pOptionValue,
                                            uint32_t optionValueLength )
 {
-    return Cellular_CommonSocketSetSockOpt( cellularHandle, socketHandle, optionLevel, option,
-                                            pOptionValue, optionValueLength );
+    CellularError_t err = CELLULAR_SUCCESS;
```

Comment:
<cellular_module>_wrapper.c file is designed to be a simple wrapper file for cellular common API.
It is designed to be generated by tool or macro replace in the future.
If the function is changed, we can move the function from cellular_r4_wrapper.c to cellular_r4_api.c

Reply:
Moved to cellular_r4_api.c.